docs/api/cli-options.md

```diff
@@ -32,7 +32,8 @@ Usage: start-storybook [options]
 | `--debug-webpack`               | Display final webpack configurations for debugging purposes<br/>`start-storybook --debug-webpack`                                                                           |
 | `--webpack-stats-json`          | Write Webpack Stats JSON to disk<br/>`start-storybook --webpack-stats-json /tmp/webpack-stats`                                                                              |
 | `--docs`                        | Starts Storybook in documentation mode. Learn more about it in [here](../writing-docs/build-documentation.md#preview-storybooks-documentation)<br/>`start-storybook --docs` |
-| `--no-manager-cache`            | Disables Storybook's manager caching mechanism. See note below.<br/>`start-storybook --no-manager-cache`                                                                    |
+| `--no-manager-cache`            | Disables Storybook's manager caching mechanism. See note below<br/>`start-storybook --no-manager-cache`                                                                     |
+| `--disableTelemetry`            | Disables Storybook's telemetry. Learn more about it [here](../configure/telemetry.md)<br/>`start-storybook --disableTelemetry`                                              |
```

Comment:
Is the CLI option really `--disableTelemetry`? Or is this a typo and it should be `--disable-telemetry`?

If it's _not_ a typo, why are we using `camelCase` instead of `kebab-case` like all the others?